assets/app/scripts/directives/logViewer.js

```diff
@@ -275,6 +275,10 @@ angular.module('openshiftConsole')
                 $timeout(function() {
                   $scope.loading = false;
                 }, 100);
+
+                if(lastLineNumber === 0) {
+                    $scope.emptyState = true;
```

Comment:
Feels like the message should be different if we get an empty log as opposed to an error. Do you know when it happens?